src/search.rs

```diff
@@ -776,7 +857,10 @@ mod tests {
                 value: S("dolor sit amet, consectetur adipiscing elit"),
                 kind: S("text"),
                 number: 10,
-                nested: Nested { child: S("second") }
+                nested: Nested { child: S("second") },
+                // TODO: This is likely a bug upstream. vectors should not be present
+                // => correct would be `_vectors: None`
+                _vectors: Some(Vectors::from(&[2000.0])),
```

Comment:
TODO(Frank): Report this upstream

Reply:
```suggestion
                nested: Nested { child: S("second") },
                _vectors: Some(Vectors::from(&[2000.0])),
```

Hey, this is not a bug; you can remove the comment